src/main/java/org/folio/rest/client/ActualCostRecordClient.java

```diff
@@ -0,0 +1,50 @@
+package org.folio.rest.client;
+
+import static org.folio.HttpStatus.HTTP_NO_CONTENT;
+
+import java.util.Map;
+
+import org.folio.rest.jaxrs.model.ActualCostRecord;
+
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.Vertx;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.ext.web.client.HttpResponse;
+
+public class ActualCostRecordClient extends OkapiClient {
+
+  public static final String ACTUAL_COST_RECORD_STORAGE = "/actual-cost-record-storage/actual-cost-records";
+
+  public ActualCostRecordClient(Vertx vertx, Map<String, String> okapiHeaders) {
+    super(vertx, okapiHeaders);
+  }
+
+  public Future<ActualCostRecord> fetchActualCostRecordById(String actualCostRecordId) {
+    return getById(ACTUAL_COST_RECORD_STORAGE, actualCostRecordId,
+      ActualCostRecord.class);
+  }
+
+  public Future<HttpResponse<Buffer>> updateActualCostRecord(ActualCostRecord actualCostRecord) {
```

Comment:
Do we really need a dedicated client for actual cost records? Maybe we can user `CirculationStorageClient` for this?